src/tsdb.h

```diff
@@ -42,33 +42,34 @@ typedef struct Series
     ChunkFuncs *funcs;
     size_t totalSamples;
     DuplicatePolicy duplicatePolicy;
-    bool isTemporary;
```

Comment:
@OfirMos Nice!!! since we don't have the `!lastDeletedSeries` we can remove `isTemporary`